src/main/java/io/airlift/units/Duration.java

```diff
@@ -164,6 +169,11 @@ public int compareTo(Duration o)
         return Double.compare(getValue(MILLISECONDS), o.getValue(MILLISECONDS));
     }
 
+    public boolean isZero()
+    {
+        return compareTo(ZERO) == 0;
```

Comment:
Why not
```java
return equals(ZERO);
```
This gets us the fast identity comparison, which seems to be the point of having a `ZERO` constant instance.